app/applicant/jobs/page.tsx

```
@@ -1,389 +1,1041 @@
 
-// Define JobFilters interface (same as in components/job-filters.tsx)
+// Flatten all tags for filtering
+const ALL_TAGS = [
+  ...TAG_CATEGORIES.WORK_MODE,
+  ...TAG_CATEGORIES.ROLES,
+  ...TAG_CATEGORIES.INDUSTRIES,
+  ...TAG_CATEGORIES.TECH_FIELDS
+];
+
+// Define filter interface
 interface JobFilters {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'JobFilters'. Consider to delete it or rename it.

'JobFilters' is defined here:


(lint/suspicious/noRedeclare)